### ISSUE TITLE
API reference does not mention the parameter quantities (grootheden)

### ISSUE THREAD

**Issue by Huite** [mbakker7/timml#106]
Coming from a question that was e-mailed to me: the API reference currently does not mention what the units or quantities are. Not having units is a feature, just like MODFLOW, as some people want to compute in m/d, ft/d, m/s, etc.

However, it would be worthwhile to add the quantities like MODFLOW does in the documentation, e.g. L/T for length over time, L^3 for volumes, etc.

**Comment by mbakker7** [comment]
It is indeed a good idea to add dimensions like L/T or L^3/T to the docs.

**Comment by Huite** [comment]
Another thing I noticed (after getting another e-mail): whether a discharge or rate is adding or removing water from the model isn't always mentioned in the docstrings.
For example, the Circular Area Sink explicitly mentions that a positive value is adding water, but a well or linesinkditch discharge (or the N argument on the inhom) should also mention it. Especially since it depends on the element unlike MODFLOW, where positive is (almost?) always adding water to the model.

**Comment by mbakker7** [comment]
You are correct. This would be another worthwhile addition to the doc strings.
